website/server/models/task.js

```diff
@@ -58,6 +58,7 @@ export let TaskSchema = new Schema({
   userId: {type: String, ref: 'User', validate: [validator.isUUID, 'Invalid uuid.']}, // When not set it belongs to a challenge
 
   challenge: {
+    name: {type: String, default: 'No Category'},
```

Comment:
"Personal"? "No challenge"?

I wonder if someone would create a challenge called "Private" or "Personal". I guess they might if they felt that was a meaningful title, or they might use a title that started with one of those words. "None" or "No challenge" might be better options to minimise the chance of confusion.
 with challenges that had